qiskit_experiments/library/randomized_benchmarking/rb_utils.py

```diff
@@ -199,6 +206,83 @@ def coherence_limit(nQ=2, T1_list=None, T2_list=None, gatelen=0.1):
 
         return coherence_limit_err
 
+    @staticmethod
+    def coherence_limit_error(
```

Comment:
We are going to deprecate `RBUtils` class since this just behaves as a scope. I think this method should be placed in somewhere else. Although I think this is not limited to RB, it would be great if this value can be automatically computed for RB based on given backend property or instmap, and populate the metadata of EPG entries. 

Currently we don't have any utils or tool collection in experiments. Perhaps it's good time to introduce something like `qiskit_experiments.utils`? @chriseclectic

Reply:
I don't think stand alone functions like this should be added to qiskit experiments at all unless they are used directly in an experiment or its analysis, and for those cases they should typically be an internal method of those classes (this also applies to all the RBUtils functions that weren't actually being used as part of RB experiment/analysis)

If this is intended to be used as part of RB experiment to construct an analysis result, it should be integrated into RB. Otherwise I don't think really think it belongs in this repo and we need to think of a better home for these sort of utility functions.

Reply:
We often compute this value to investigate the error budget. It might be good to compute the error limit with this function as a metadata of EPG entries (as IRB compute theoretical error bounds). But this function can be used more widely to discuss lower error bound. In that sense perhaps this can be move to terra?